lib/blockscore/base.rb

```diff
@@ -60,8 +60,16 @@ def persisted?
       !id.nil? && !deleted?
     end
 
+    def deleted?
```

Comment:
I don't think this is sufficient. That is an inference on my part, but running it without the VCR_RECORD_MODE produces misinformation. My inference is this is not taken into account in the diff above. I do support running mutant, but only if it generates something actionable.